packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -94,17 +103,27 @@ cp -pa .%{gem_dir}/* \
 %{gem_libdir}
 %{gem_instdir}/locale
 %exclude %{gem_cache}
+%exclude %{gem_instdir}/.eslintrc
+%exclude %{gem_instdir}/.stylelintrc
```

Comment:
These RC files are not part of the gem, you can drop those excludes. They will probably fail the build.